content/rust/github/publish-github-release-multi-arch.md

```diff
@@ -0,0 +1,419 @@
+---
+date: 2000-01-01T00:00:00+00:00
+title: Create multi-Architecture GitHub releases for Rust projects
+author: KN4CK3R
+draft: true
+image: https://dummyimage.com/650x300/253d5f/ffffff
+keywords:
+- rust
+- github
+tags:
+- rust
+- githug
+- arm
+- arm64
+- amd64
+---
+
+This guide will walk you through the steps required to build multi-architecture Rust projects and publish the binaries on GitHub.
+This guide assumes you have a basic understanding of Rust, Docker and GitHub.
+
+{{< alert info >}}
+To cross compile Rust projects this tutorial uses the sample Rust project and the Cross-Compilation Docker image presented in the tutorial [Publishing Rust Docker Images for ARM](...).
```

Comment:
Here I need help because I don't know how to reference an other page.

Reply:
I think https://gohugo.io/content-management/cross-references/ should explain references to other pages well enough?